apps/ui/src/views/Swap/context.ts

```diff
@@ -77,15 +76,17 @@ const useSwap = () => {
     if (!tokenA || !tokenB) {
       return SwapMode.CrossChainOrder;
     }
-    if (isEthAsset(tokenA)) {
-      if (isCkbNativeAsset(tokenB)) {
+    if (EthModel.isCurrentChainAsset(tokenA)) {
```

Comment:
There may be some redundant judging, but I think it's clearer to write the code this way

```ts
if (
  EthModel.isCurrentChainAsset(tokenA)
  && CkbModel.isNativeAsset(tokenB)
) {
  return SwapMode.CrossChainOrder;
}

if (
  EthModel.isCurrentChainAsset(tokenA) 
  && EthModel.isShadowEthAsset(tokenB) && 
  && EthModel.equals(tokenA, tokenB)
) {
  return SwapMode.CrossIn;
}
//. ..